test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3452,6 +3454,7 @@ def forward(self, input_mask, some_const):
         self.run_test(MaskedFillModel(), (mask, constant))
 
     @skipIfUnsupportedMinOpsetVersion(11)
+    @disableScriptTest()
```

Comment:
How about this test? Is this going to e addressed?

Reply:
yes this is also being addressed in #46866